common/web/gesture-recognizer/src/engine/touchEventEngine.ts

```diff
@@ -226,6 +227,8 @@ export class TouchEventEngine<ItemType, StateToken = any> extends InputEventEngi
         this.inputStartSignalMap.set(touchpoint, eventSignalPromise);
 
         return eventSignalPromise.corePromise;
+      } else {
+        return Promise.resolve();
```

Comment:
Is this a bug fix?

Reply:
It is not.  An async method implicitly returns a Promise when its end is reached... but the `noImplicitReturns` flag prevents us from utilizing this.

Reply:
Again, the closure here is not an `async` method, so it could have been returning `undefined` previously. I don't _think_ this changes behaviour because it would have been falling back to `defaultWaitFactory()` but I haven't dug deep enough to be 100% clear on that.